libr/core/canal.c

```diff
@@ -1880,12 +1886,12 @@ static int core_anal_graph_nodes(RCore *core, RAnalFunction *fcn, int opts, PJ *
 		pj_end (pj);
 		pj_end (pj);
 	}
-	free (pal_jump);
-	free (pal_fail);
-	free (pal_trfa);
-	free (pal_curr);
-	free (pal_traced);
-	free (pal_box4);
+	R_FREE (pal_jump);
+	R_FREE (pal_fail);
```

Comment:
No need to R_FREE before return

Reply:
If they are not freed, Coverity triggers an Resource leak issue.